app/models/energy/constructionabridged.py

```diff
@@ -0,0 +1,91 @@
+"""Construction Abridged Schema"""
+from pydantic import BaseModel, Schema, validator, ValidationError, constr
+from typing import List, Union
+from enum import Enum
+from uuid import UUID, uuid4
+from datetime import datetime
+
+
+class EnergyConstructionWindowAbridged(BaseModel):
```

Comment:
I just realized that I have been naming all construction objects  in the Python code without the use of the word "Energy" and I didn't realize it was different here.
With EnergyMaterials, I realize we wanted to include the word "Energy" to make it clear that we were talking about a different type of material than a radiance material.  However, we don't have the same restriction with Constructions. 

Given this, can we change the names to the following:
`OpaqueConstruction`, `OpaqueConstructionAbridged`, `WindowConstruction`, `WindowConstructionAbridged`.

Hopefully, it should just be a quick Find and Replace, @tanushree04